modules/olm-pruning-index-image.adoc

```diff
@@ -123,10 +123,16 @@ $ opm index prune \
     -f {index-image-pullspec} \// <1>
     -p {package1},{package2},{package3} \// <2>
     -t <target_registry>:<port>/<namespace>/{index-image} <3>
+    -i registry.redhat.io/openshift4/ose-operator-registry/{index-image} 
```

Comment:
should there be a `<4>` on this line and an explanation below? Also if this is only needed for P & Z, what's the default `-i` value for amd64?

Reply:
If that line isn't needed for other architectures I would suggest we remove the note and make it a callout. 

<4> Use `-i` only with IBM Power Systems and IBM Z images.

Reply:
The **default value of base image is** `quay.io/operator-framework/upstream-opm-builder` which isn't multi-arch and thus prune command - `opm index prune ` fails on Power ( should fail on Z too , yet to be validated though) . To resolve this had to replace it with - `registry.redhat.io/openshift4/ose-operator-registry` which is multi-arch passed using flag `-i` to prune command ....  fyi @SNiemann15  @nickboldt

Reply:
So... leave it in with the default?
```
-i quay.io/operator-framework/upstream-opm-builder <4>
```

then below...